src/nimblepkg/packageinstaller.nim

```diff
@@ -62,14 +64,14 @@ proc setupBinSymlink*(symlinkDest, symlinkFilename: string,
       if fixChcp:
         contents.add "chcp 65001 > nul && "
       else: contents.add "chcp 65001 > nul\n@"
-    contents.add "\"" & symlinkDest & "\" %*\n"
+    contents.add "\"%~dp0\\" & symlinkDestRel & "\" %*\n"
```

Comment:
damn, this is some magic. You might want to consider using raw string literals here.

Reply:
Raw strings don't help since the string contains `"` and `\n`. Doesn't look any prettier.